vpr/src/place/place.cpp

```diff
@@ -3037,7 +3041,7 @@ void transform_blocks_affected(t_pl_blocks_to_be_moved blocksAffected) {
 }
 
 #ifdef VTR_ENABLE_DEBUG_LOGGING
```

Comment:
Minimize added code in try_swap (already a critical routine and overly long).  Create high-level functions, if possible in a new place_debug.cpp (or some such) file and call them from here.  Should only see a few calls / condition checks in try_swap for breakpoints and they should be well commented.

Reply:
Looking closer, I think Mahshad already did this.